tokens/src/core/components/Code.json

```diff
@@ -1,32 +1,33 @@
 {
   "typography": {
     "code": {
-      "font-size": { "value": "87.5%", "type": "percentage", "source": "$code-font-size" },
+      "font-size": { "source": "$code-font-size", "$value": "87.5%", "$type": "percentage" },
```

Comment:
[suggestion] I believe this `87.5%` is equivalent to the `{typography.font.size.sm}` if we wanted to treat this token an alias to a semantic token (i.e., if/when `{typography.font.size.sm}` changes, this token will also change).

Reply:
[inform] Implemented in follow-up PR: https://github.com/openedx/paragon/pull/3222

Reply:
A small difference exists between `87.5%` and `0.875rem`. The `87.5%` value is calculated relative to the font size of the parent element. For example, if the parent has a font size of `16px`, then `87.5%` will be equivalent to `14px`. However, if the parent font size is different, the resulting pixel value will change accordingly.

In contrast, `0.875rem` is always calculated relative to the root element’s (`<html>`) base font size, which is typically `16px` by default in browsers. Thus, `0.875rem` will consistently be equivalent to `14px` as long as the html font size is `16px`, regardless of the parent element’s font size.

Currently, when using the `0.875rem` value from the token `{typography.font.size.sm}`, the value is indeed calculated relative to the root font size of the html tag (`16px` = `1rem`). The previous percentage value of `87.5%` was calculated based on the parent font size of `1.125rem` (the font size of the body).

I suggest using `87.5%`, what do you think?

Reply:
[inform] Implemented in follow-up commit: [6676868 (#3186)](https://github.com/openedx/paragon/pull/3186/commits/6676868e6c46d70d91f0bdc42b7c2848e3b0aa7c)